internal/uid/uid.go

```diff
@@ -0,0 +1,16 @@
+package uid
+
+import (
+	"os"
+	"strconv"
+)
+
+// GetRootlessUID returns the UID of the user in the parent userNS
+func GetRootlessUID() int {
```

Comment:
Please call this from `pkg/blobinfocache/default.go`, and drop the copy in that file.